src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,11 @@
+package course_project;
+
+import course_project.services.SeaBattle;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
It seems indentation is shifted
![image](https://user-images.githubusercontent.com/58872509/134234647-69ab186e-ffd0-4818-84fc-3def141659eb.png)

Reply:
can you describe this case in detail?
my view:
<img width="722" alt="Screen Shot 2021-09-21 at 10 28 39 PM" src="https://user-images.githubusercontent.com/26281676/134235335-81ef33d2-923a-4443-83f1-4451893c2608.png">

Reply:
Well done! The game interface looks pretty nice, even though it lacks row delimiters. Playing is comfortable, it shows pretty informative messages. Also, a lot of features are introduced, as well as exit code and description. You show good knowledge of Java Core and Java libraries, abstractions are mostly well-picked, architecture is good, but it can be improved. Also, you have lots of static, and it's not recommended. 

Approved!

Reply:
I understood you. Thanks for the review!